docs/core/resource.rst

```diff
@@ -198,7 +198,7 @@ As an example, let's build a code that will copy a collection into another:
     from pyramid import Configurator
 
 
-    config = Configurator(settings=DEFAULT_SETTINGS)
+    config = Configurator(settings={**DEFAULT_SETTINGS})
```

Comment:
Is a shallow copy here enough? Maybe it's a good idea to note with XXX that this is a bug on pyramid.